CRM/Contribute/Form/Contribution.php

```diff
@@ -650,8 +669,14 @@ public function buildQuickForm() {
       $paymentInstrument = $this->add('select', 'payment_instrument_id',
         ts('Payment Method'),
         array('' => ts('- select -')) + CRM_Contribute_PseudoConstant::paymentInstrument(),
-        TRUE, array('onChange' => "return showHideByValue('payment_instrument_id','{$checkPaymentID}','checkNumber','table-row','select',false);")
+        TRUE
       );
+      $this->addField('card_type', array('entity' => 'FinancialTrxn', 'context' => 'get'));
+      $this->add('text', 'pan_truncation', ts('Card Number'), array(
+        'size' => 5,
+        'maxlength' => 4,
+        'autocomplete' => 'off',
+      ));
     }
```

Comment:
@KarinG I think this is a use case where you are recording in Civi that you have received a credit card payment not through Civi & you have that detail